src/devices/video/gb_lcd.h

```diff
@@ -262,6 +256,8 @@ class sgb_ppu_device : public dmg_ppu_device
 	virtual void update_sprites() override;
 	virtual void update_scanline(uint32_t cycles_to_go) override;
 	void refresh_border();
+	
+	void sgb_vram_memcpy(uint8_t* dst, uint8_t start, size_t num_tiles);
```

Comment:
Note that just above this, there's a parameter declared as `uint8_t *data` - please follow that style (it's closer to how the C++ parser handles it too, as the unary `*` here is right-associative).